.github/workflows/action-image-scan-deploy-dashing.yaml

```diff
@@ -21,5 +21,5 @@ jobs:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
     
-    - run: docker build --build-arg ROS_DISTRO=$ROS_DISTRO -t oriserobotics/ros-$ROS_DISTRO:devel .
-    - run: docker push oriserobotics/ros-$ROS_DISTRO:devel
+    - run: docker build --build-arg ROS_DISTRO="$ROS_DISTRO" -t oriserobotics/ros-"$ROS_DISTRO":devel .
```

Comment:
Curiosity, why the need for double quotes(") here?

Reply:
Super Linter now has an [`actionlint`](https://github.com/orise-robotics/ros_ws/runs/4686218184?check_suite_focus=true#step:4:188) which runs `shellcheck` against the shell commands. The error pointed is 
```
SC2086:info:1:70: Double quote to prevent globbing and word splitting
```